src/test/java/com/documents/InterruptionRequestDocumentTests.java

```diff
@@ -0,0 +1,216 @@
+    }
+
+
+    public void setup(InterruptionRequestDocument interruptionRequestDocumentToSet) {
+
+        interruptionRequestDocumentToSet.setId((long) 3);
+        interruptionRequestDocumentToSet.setDocumentName("Document_Name");
+        interruptionRequestDocumentToSet.setStudentName("Student_Name");
+        interruptionRequestDocumentToSet.setCurrentYear(2);
+        interruptionRequestDocumentToSet.setYearOfStudy(2017);
+        interruptionRequestDocumentToSet.setStartingSemester(2);
+        interruptionRequestDocumentToSet.setFromUniversityYear(2018);
+        interruptionRequestDocumentToSet.setNumberOfSemesters(2);
+
+    }
+
+    @Test
+    public void behavioural_interruptionRequest_save_should_return_true() throws Exception {
+
+        //Act
+        InterruptionRequestDocument interruptionRequestDocumentAfterSave = new InterruptionRequestDocument();
+        InterruptionRequestDocument interruptionRequestDocumentToSave = new InterruptionRequestDocument();
+
+        setup(interruptionRequestDocumentToSave);
+
+        when(interruptionRequestDocumentRepository.save(any(InterruptionRequestDocument.class))).thenReturn(interruptionRequestDocumentAfterSave);
+
+        interruptionRequestDocumentAfterSave = interruptionRequestDocumentServiceImpl.save(interruptionRequestDocumentToSave);
+
+        //Assert
+        assertNotNull(interruptionRequestDocumentAfterSave);
+    }
+
+    @Test
+    public void functionality_interruptionRequest_save_should_return_interruptionRequest() throws Exception {
+
+        //Act
+        InterruptionRequestDocument interruptionRequestDocumentAfterSave = new InterruptionRequestDocument();
+
+        setup(interruptionRequestDocumentAfterSave);
+
+        when(interruptionRequestDocumentRepository.save(any(InterruptionRequestDocument.class))).thenReturn(interruptionRequestDocumentAfterSave);
+
+        InterruptionRequestDocument savedInterruptionRequest = interruptionRequestDocumentServiceImpl.save(interruptionRequestDocumentAfterSave);
+
+        //Assert
+        assertEquals(Long.valueOf(3), savedInterruptionRequest.getId());
+        assertEquals("Document_Name", savedInterruptionRequest.getDocumentName());
+        assertEquals("Student_Name", savedInterruptionRequest.getStudentName());
+        assertEquals(Integer.valueOf(2), savedInterruptionRequest.getCurrentYear());
+        assertEquals(Integer.valueOf(2017), savedInterruptionRequest.getYearOfStudy());
+        assertEquals(Integer.valueOf(2), savedInterruptionRequest.getStartingSemester());
+        assertEquals(Integer.valueOf(2018), savedInterruptionRequest.getFromUniversityYear());
+        assertEquals(Integer.valueOf(2), savedInterruptionRequest.getNumberOfSemesters());
+
+    }
+
+    @Test
+    public void behavioural_interruptionRequest_findById_should_return_true() throws Exception {
+
+        //Act
+        InterruptionRequestDocument interruptionRequestDocumentToSave = new InterruptionRequestDocument();
```

Comment:
rename interruptionRequestDocumentToSave into interruptionRequestDocumentToFind  .. you are finding, not saving something.